ldms/src/contrib/sampler/gpu_metrics_sampler/gather_gpu_metrics.cpp

```diff
@@ -0,0 +1,154 @@
+ *      contributors may be used to endorse or promote products derived
+ *      from this software without specific prior written permission.
+ *
+ *      Modified source versions must be plainly marked as such, and
+ *      must not be misrepresented as being the original software.
+ *
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+
+#include "gather_gpu_metrics_from_one_api.h"
+#include <level_zero/zes_api.h>
+
+#include <iostream>
+#include <iomanip>
+#include <chrono>
+
+using namespace std::chrono;
+using namespace std;
+
+
+void printGpuMetrics(ze_device_handle_t device, uint32_t devNumber) {
+    cout << " " << devNumber << "    -1 Format: Value          Descriptive text" << endl;
+
+    uint32_t metricNumber = 0;
+    cout << fixed << setprecision(2);
+    cout << " " << devNumber << "     " << metricNumber++ << "         "
+         << getGpuUtilization(device) << "           gpu_util (%)" << endl;
+
+    cout << " " << devNumber << "     " << metricNumber++ << "         "
+         << getMemoryUtilization(device) << "           mem_util (%)" << endl;
+
+    cout << " " << devNumber << "     " << metricNumber++ << "         "
+         << getMemVRAMUsed(device) << "       mem_vram_used" << endl;
+
+    cout << " " << devNumber << "     " << metricNumber++ << "         "
+         << getSysClockFreq(device) << "           sys_clock_freq (MHz)" << endl;
+
+    cout << " " << devNumber << "     " << metricNumber++ << "         "
+         << getMemoryReadBandwidth(device) << "          mem_read_bandwidth (kilobaud)" << endl;
+
+    cout << " " << devNumber << "     " << metricNumber++ << "         "
+         << getMemoryWriteBandwidth(device) << "          mem_write_bandwidth (kilobaud)" << endl;
+
+    cout << " " << devNumber << "     " << metricNumber++ << "         "
+         << getPerfLevel(device) << "           perf_level" << endl;
+
+
+    cout << " " << devNumber << "     " << metricNumber++ << "         "
+         << getPowerUsage(device) << "             power_usage (mW)" << endl;
+
+    //  The following is no longer supported.
+//    cout << " " << devNumber << "     " << metricNumber++ << "         "
+//         << getPowerCap(device) << "         power_cap (mW)" << endl;
```

Comment:
Please remove code that is commented out

Reply:
Removed the commented code